### ISSUE TITLE
questions about model setup

### ISSUE THREAD

**Issue by bbolker** [wzmli/undercount#1]
These are mostly for @wzmli.  **tl;dr** (1) Some of the model-building decisions are weird, or I'm misunderstanding them. (2) It seems to make sense for this application to err on the side of simplicity, i.e. a straight discrete-time SIR model. I don't expect any of these changes to make much difference in the results, but it would be good to clarify/simplify.

  * you originally had beta = gamma = 0.3; this doesn't make any sense to me since it implies R0 = 1 (unless there is some other scaling I'm not thinking about???) -- I changed beta to 0.35

  * it would be a little bit easier to interpret this if we set `do_hazard = FALSE` (e.g., I'm computing R0 as beta/gamma, but it should really be (1-exp(-beta))/(1-exp(-gamma)) if we have hazard correction on ... ?

  * in parameters.R you set c_delay_cv to 1 and c_delay_mean to 0.5; do these values make sense? 

[I now see that although these parameters don't appear in the model definition, they parameters are used by default in add_conv() -- I'm not sure I like this design, seems a little too 'magic'? -- but that's a MacPan question, not for here]

  * you have `disp = 1000`, which is basically a very low-noise setting. Would it be simpler/better to do a deterministic simulation?

In other words, we could do a deterministic SIR without hazard correction, leave the explicit reporting component out, and just multiply daily incidence by `c_prop`.

**Comment by wzmli** [comment]
I have now simplified the model to make it less magical. Close the issue if you think it is ok

**Comment by bbolker** [comment]
Nice!  You could   `%>% add_sim_report_expr('reports', ~ c_prop * (S_to_I) * (S))` instead of computing it downstream (or `~ c_prop * incidence` ? I don't know if you can use derived variables like that).  But that's getting **really** picky (and is entirely stylistic, makes no difference to the results).
